python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -521,9 +556,10 @@ def run(self, *, exposures, id_generator=None, result=None):
 
         result.psf_stars_footprints, result.background, candidates = self._compute_psf(result.exposure,
                                                                                        id_generator)
-        result.psf_stars = result.psf_stars_footprints.asAstropy()
+        self._normalize_calibration_flux(result.exposure, result.psf_stars_footprints)
+        self._measure_aperture_correction(result.exposure, result.psf_stars_footprints)
 
-        self._measure_aperture_correction(result.exposure, result.psf_stars)
+        result.psf_stars = result.psf_stars_footprints.asAstropy()
```

Comment:
Huh... good catch! I am rather surprised that worked! I guess `self.measure_aperture_correction.run` only cares that the input duck types as a catalog in the most generic manner?

Reply:
I think so, but I'm also surprised that it worked and filled in the columns.  Certainly wouldn't be naively expected to work.